modules/ROOT/pages/dataweave-cookbook-defaults.adoc

```diff
@@ -15,48 +17,50 @@ include::partial$dataweave1-links.adoc[tag=dataweave1Examples]
 
 == Using the Key Word default
 
-This example uses `default` to set a default value for a payload in case
-the payload value is `null`.
+Consider a Mule application that expects a JSON input with fields `id` and `name` to do a transformation. +
```

Comment:
I aski
```suggestion
Consider a Mule application that expects a JSON input with fields `id` and `name` to do a transformation. 
```

Reply:
I asked Kim about that `+`, and looks like it goes against our normal sentence style because it prevents normal wrapping.

Reply:
Yes, GLOBALLY avoid using a space followed by "+" to force a line break, because it actually creates a paragraph break (which you want to create by pressing `return` twice, rather than using the Asciidoctor syntax +).
Carefully consider where you break for a new paragraph. Such a break should signal the transition to a new focus.

Reply:
GLOBALLY avoid using "do" when you really mean "perform":
```suggestion
Consider a Mule application that expects a JSON input that contains the fields `id` and `name` to perform a DataWeave transformation, but the value for these fields is `null`. Configuring default values for these fields enables the transformation to occur:
```
Is this technically accurate? I got confused by us saying that these fields have the value `null` but we configure defaults that are actually also null: 0000 and Undefined. Does providing these values prevent an error or failure that would be caused by the value `null`?

Reply:
When I read below, we don't actually mention a value of `null` but rather "an empty message". Which is correct, if they are the same thing?
Also, why do we give an example of fields that actually have values if this text is about providing defaults when they do not? If you want to show the example with fields, I suggest starting this topic with "Consider a Mule application that expects JSON input containing values for the fields `id` and `name`:", and then show the "normal" result. THEN mention the issue of what happens if there are NO values for these fields, and how providing defaults solves this.

Reply:
I would not make these bullet list items, but a running narrative from stating the normal situation, with values, and then describing the less normal situation and its solution by providing defaults.